docs/en/stack/ml/nlp/ml-nlp-inference.asciidoc

```diff
@@ -197,6 +198,10 @@ POST _reindex
 However, those web log messages are unlikely to contain enough words for the
 model to accurately identify the language.
 
+TIP: When using deployed NLP models, set the reindex `size` option to a value
```

Comment:
Since this page is entirely about NLP models, I don't think this introductory qualifier adds any value:
```suggestion
TIP: Set the reindex `size` option to a value
```